### DISCUSSION TITLE
Changing the layout

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [GetPublii/Publii#1454] [Help] [answered]
I am guessing I need to change an actual template somewhere but I thought I would ask first.

When I preview the site, I get the following:
![image](https://github.com/GetPublii/Publii/assets/111164245/4435cdb9-624d-4d17-995d-1903caeee09c)

If I scroll down the menu background changes as below:
![image](https://github.com/GetPublii/Publii/assets/111164245/e3aa3333-2d94-4134-97f6-053353078e7f)

What I am wanting to achieve is for the page to always look as follows:
<img width="926" alt="image" src="https://github.com/GetPublii/Publii/assets/111164245/db85394b-bee6-4b9a-bb96-e172b7fd033a">

Do I need to manipulate with CSS or can I change it at a more fundamental level?

**Comment by ZuluPilgrim** [comment]
Doing it with CSS overides

**Comment by candidexmedia** [comment]
Hi there! Which theme are you using, and how comfortable are you with making changes in code?

I've done this before in mercury, and it could be achieved by changing some css, or removing some JS.

Edit: I see that you've made other posts with questions about theme editing, and looks like you've figured it out! 🙂 

General advice for theme editing: find your Site directory, go to your `input > theme` folder. You can make direct CSS changes in the main.css file OR make overrides in the app. JS files may have to involve editing both the regular file and min file. I'd suggest reading the dev docs on theme overrides too so that your changes are kept if you ever have to update the theme (and, tbh, reading the dev docs top to bottom doesn't hurt).

See (general CSS editing advice): https://github.com/GetPublii/Publii/discussions/1412#discussioncomment-8696413

**Reply by ZuluPilgrim** [reply]
Hi, I am using Simple (v.2.8.0).

Right now I am using the custom CSS plugin to make changes in the CSS.

**Reply by ZuluPilgrim** [reply]
Last question on this.

I am trying to change the colour of the mobile menu bars but can't work out what is the selector code I need to use to reference this.

![image](https://github.com/GetPublii/Publii/assets/111164245/47f4448c-74f5-4bf9-8a22-c94e2957ea4d)

**Reply by candidexmedia** [reply]
Checking in @ZuluPilgrim : Have you solved this?

**Comment by ZuluPilgrim** [comment]
Below is what the code looks like.

I have tried a number of options e.g. #js-header > nav > button, but to no avail.

```
<nav class="navbar js-navbar">
   <button class="navbar__toggle js-toggle" aria-label="Menu" aria-haspopup="true" aria-expanded="false"><span class="navbar__toggle-box"><span class="navbar__toggle-inner">Menu</span></span></button>
   <ul class="navbar__menu">
      <li><a href="https://demo.site.com/meet-niki/" target="_self">Meet Niki</a></li>
      <li><a href="https://demo.site.com/services/" target="_self">Services</a></li>
      <li><a href="https://demo.site.com/pricing/" target="_self">Pricing</a></li>
      <li><a href="https://demo.site.com/contact-us/" target="_self">Contact Us</a></li>
   </ul>
</nav>
```

**Reply by candidexmedia** [reply]
Can you share a live page?

**Comment by ZuluPilgrim** [comment, marked as answer]
Apologies, I forgot I had also asked this there: [https://github.com/GetPublii/Publii/discussions/1460](https://github.com/GetPublii/Publii/discussions/1460). 

I worked out a solution discuss there.

**Reply by candidexmedia** [reply]
Sweet!
